Formula/r/rpm.rb

```diff
@@ -14,39 +14,42 @@ class Rpm < Formula
     regex(/href=.*?rpm[._-]v?(\d+\.\d+(?:\.(?:\d|[1-8]\d+)(?:\.\d+)*))\.t/i)
   end
 
-  bottle do
-    sha256 x86_64_linux: "84a6f8905a17d3797cd84a3d38874b951c7c00418fba684a6763022dc0119b8e"
-  end
+  # bottle do
+  #   sha256 x86_64_linux: "84a6f8905a17d3797cd84a3d38874b951c7c00418fba684a6763022dc0119b8e"
+  # end
```

Comment:
Please do not adjust the bottle block
```suggestion
  bottle do
    sha256 x86_64_linux: "84a6f8905a17d3797cd84a3d38874b951c7c00418fba684a6763022dc0119b8e"
  end
```